jdaviz/app.py

```diff
@@ -845,7 +845,7 @@ def get_subsets(self, subset_name=None, spectral_only=False,
         spectral_only : bool
             Return only spectral subsets.
         spatial_only : bool
-            Return only spatial subsets.
+            Return only spatial subsets, except masked subset state (this is unusual).
```

Comment:
I'm worried this can read as "its unusual that this doesn't return masked subset states".... how about the following?

```suggestion
            Return only spatial subsets, except masked subsets (uncommon).
```